docs/user/dashboard/dashboard.asciidoc

```diff
@@ -0,0 +1,472 @@
+
+[float]
+[[vega]]
+=== Create custom panels with Vega
+
+Build custom visualizations using *Vega* and *Vega-Lite*, backed by one or more data sources including {es}, Elastic Map Service, 
+URL, or static data. Use the {kib} extensions to embed *Vega* in your dashboard, and add interactive tools.
+
+Use *Vega* and *Vega-Lite* when you want to create a visualization for:
+
+* Aggregations that use `nested` or `parent/child` mapping
+* Aggregations without an index pattern
+* Queries that use custom time filters
+* Complex calculations
+* Extracting data from _source instead of aggregations
+* Scatter charts, sankey charts, and custom maps
+* Using an unsupported visual theme
+
+[role="screenshot"]
+image::images/vega.png[Vega UI]
+
+*Vega* and *Vega-Lite*  are declarative formats that:
+
+* Create complex visualizations
+* Use JSON and a different syntax for declaring visualizations
+* Are not fully interchangeable
+
+For more information about *Vega* and *Vega-Lite*, refer to:
+
+* <<vega-lite-tutorial-create-your-first-visualizations,Create your first visualization>>
+* <<vega-tutorial-update-kibana-filters-from-vega,Update Kibana filters>>
+* <<reference-for-kibana-extensions,Reference for Kibana extensions>>
+* <<resources-and-examples,Resources and examples>>
+
+[float]
+[[timelion]]
+=== Create panels with Timelion
+
+*Timelion* is a time series data visualizer that enables you to combine independent data sources within a single visualization. 
+
+*Timelion* is driven by a simple expression language that you use to:
+
+* Retrieve time series data
+* Perform calculation to tease out the answers to complex questions
+* Visualize the results
+
+[role="screenshot"]
+image::images/timelion.png[Timelion UI]
+
+Ready to try out Timelion? For step-by-step tutorials, refer to:
+
+* <<timelion-tutorial-create-time-series-visualizations,Create time series visualizations>>
+* <<timelion-tutorial-create-visualizations-with-mathematical-functions,Create visualizations with mathematical functions>>
+* <<timelion-tutorial-create-visualizations-withconditional-logic-and-tracking-trends,Create visualizations with conditional logic and tracking trends>>
+
+[float]
+[[save-panels]]
+=== Save panels
+
+When you’ve finished making changes, save the panels.
+
+. Click *Save*.
+. Add the *Title* and optional *Description*.
+. Click *Save and return*.
+
+[float]
+[[add-existing-panels]]
+== Add existing panels
+
+Add panels that you’ve already created to your dashboard. 
+
+On the dashboard, click *Add an existing*, then select the panel you want to add.
+
+When a panel contains a stored query, both queries are applied.
+
+[role="screenshot"]
+image:images/Dashboard_add_visualization.png[Example add visualization to dashboard]
+
+To make changes to the panel, put the dashboard in *Edit* mode, then select *Edit visualization* or *Edit panel title* from the panel menu. 
+The changes you make appear in every dashboard that uses the panel.
```

Comment:
This is a tough one. The name of the button actually changes depending on the type of panel you are editing. It could be Edit Map, or Edit Lens, as well. I'm not sure how best to capture that, but it could cause confusion.

Additionally, the changes that you make through the 'Edit' button mentioned above will appear in every dashboard that uses the panel, but changes made via 'Edit panel title' will only appear in the dashboard you made the change in. This should be clarified.